### DISCUSSION TITLE
How to show the logout confirmation page with keycloak-js adapter

### DISCUSSION THREAD

**Discussion by eliza-budeanu** [keycloak/keycloak#24426] [Q&A]
I've seen some discussions around the possibility to display the confirmation logout page as an option - either in the server configuration logout options or as a skipIdTokenHint parameter to the logout method from the  keycloak.js which will cause adapter to not send id_token_hint to Keycloak during logout and hence enforce that logout confirmation screen is displayed.

Is there any plan for implementing it or is there a different way to choose to show the logout confirmation page if we are using the Keycloak.js adapter?

Looking trough the adapter it looks like it's always attaching the id_token_hint if id_token is present but not sure if I'm looking at the right place :)

https://github.com/keycloak/keycloak/blob/1145568324d07102da5fbc1d92f5de6a122b6bef/js/libs/keycloak-js/src/keycloak.js#L494

**Comment by jonkoops** [comment]
I am not sure whether this is currently implemented in Keycloak JS, taking a quick glance at the code I don't think it is. If there is an existing feature in Keycloak that allows this I think we should add support to Keycloak JS as well. Feel free to submit this as an feature request in our issue tracker.
